tests/data/cases/preview_import_line_collapse.py

```diff
@@ -0,0 +1,13 @@
+# flags: --preview
```

Comment:
Could you add some more test cases, e.g. a comment line right after the import? Not every test case needs to be in its own file.

Reply:
sure thing! I added a few more test cases to this file and the other one, let me know if you think there should be any other test cases covered!

Reply:
also modified the logic to collapse to one line with comments as well since removing that condition didn't affect any existing test cases